### ISSUE TITLE
Migrations syntax mistated on docs page

### ISSUE THREAD

**Issue by djfurman** [sdispater/orator#228]
Hi, I think your project is awesome!

I love Laravel and have grown to love Eloquent as an ORM, but prefer python for development especially as I work more with serverless applications on AWS; so I couldn't be more excited seeing a pythonic Eloquent 💯 

I found an issue in the [homepage](https://orator-orm.com/) docs as I'm working on my first real attempt at orator. I forked the repository, but the docs must be maintained elsewhere.

If you can point me to them I'm happy to PR them for you.

# Scenario

As a user getting started with `orator-orm`
When I install orator locally
And generate a migration using the syntax shown on the [homepage](https://orator-orm.com/) at the `h1` for Migrations
Then I should not receive syntax errors

## Re-produce

Given that the current text shows `orator migrations:make create_users_table`
When I attempt to run this generator locally,
Then I receive the following error

```
  [NamespaceNotFound]
  There are no commands defined in the "migrations" namespace.
```

## Desired

Given that the current text shows `orator make:migration create_users_table`
When I attempt to run this generator locally,
Then the generator greats the migrations directory along with the `yyyy_mm_dd_hhmmss_[migration_name].py` file

Also, for getting started with `orator-orm` the `orator model:make -m User` command may be a good showcase since it scaffolds the model and migration and is (for me a least) my most common migration generator in my workflow.

**Comment by josephmancuso** [comment]
@djfurman yes I see that. I'm pretty sure that the home page is not in the docs. We'll get back to you on that.

@sdispater is the homepage somewhere specific? i.e specific to the documentation tool you are using maybe?
